### ISSUE TITLE
Bug: compat anyio version validation fails with anyio version 4.6.2.post1

### ISSUE THREAD

**Issue by yann-combarnous** [airtai/faststream#1852]
**Describe the bug**
In faststream `_compat.py`, anyio major validation is done using:
```
anyio_major, *_ = map(int, get_version("anyio").split("."))
```

This fails with anyio latest version `4.6.2.post1`

**How to reproduce**
Install latest anyio version.

**Suggested fix**
Use

```
anyio_major = int(get_version("anyio").split(".")[0])
```

**Comment by dotX12** [comment]
@yann-combarnous fixed in https://github.com/airtai/faststream/releases/tag/0.5.27

**Comment by Lancetnik** [comment]
Closed as copy of #1849
